node/src/Utils.h

```diff
@@ -283,6 +312,23 @@ namespace node {
         return true;
     }
 
+    inline bool GruOperators(const Napi::Value& jsValue,
+                             ml::GruOperators& gruOperators,
```

Comment:
```
"gru operators": {
    "category": "structure",
    "members": [
      {"name": "reset gate activation", "type": "operator", "optional": true},
      {"name": "new gate activation", "type": "operator", "optional": true}
    ]
  },
```
Define the  `GruOperators` struct in webnn.json to `OperatorArray` object
```
 "operator array": {
    "category": "object",
    "methods": [
      {
        "name": "size",
        "returns": "size_t"
      },
      {
        "name": "set",
        "returns": "void",
        "args": [
          {"name": "operand", "type": "operand"}
        ]
      }
    ]
  },
```

Reply:
Replaced `gru operators` with `operator array`. PTAL, thanks!